pallets/msa/src/lib.rs

```diff
@@ -154,13 +168,22 @@ impl<T: Config> Pallet<T> {
 	}
 
 	pub fn add_key(msa_id: MessageSenderId, key: &T::AccountId) -> DispatchResult {
-		<KeyInfoOf<T>>::try_mutate(key, |maybe_msa| {
+		KeyInfoOf::<T>::try_mutate(key, |maybe_msa| {
 			ensure!(maybe_msa.is_none(), Error::<T>::DuplicatedKey);
 
 			*maybe_msa =
 				Some(KeyInfo { msa_id, expired: T::BlockNumber::default(), nonce: Zero::zero() });
 
-			Ok(())
+			// adding reverse lookup
```

Comment:
leftover comment

Reply:
that's just a small description comment for anyone to know why we have this second mutate inside